src/js/msp/MSPHelper.js

```diff
@@ -24,7 +24,8 @@ function MspHelper () {
     'TELEMETRY_IBUS': 12,
     'IRC_TRAMP': 13,
     'RUNCAM_DEVICE_CONTROL': 14, // support communitate with RunCam Device
-    'LIDAR_TF': 15
+    'LIDAR_TF': 15,
+    'FRSKY_OSD': 16
```

Comment:
We are trying to unify a little the code with Sonar rules, you can look at the issues of your PR here https://sonarcloud.io/project/issues?id=betaflight_betaflight-configurator&pullRequest=1853&resolved=false&types=CODE_SMELL, but the only one is to add a trailing comma to the latest element of this list 😉

Reply:
@McGiverGim I do prefer to always add the trailing comma, but since I wasn't aware you were in the process of changing that, I decided to keep the existing style of that declaration. Thanks for the clarification! I'll amend it right now.